hardware/Drivetrain.java

```diff
@@ -92,13 +93,43 @@ public void init(HardwareMap hwMap) {
     }
 
     public void teleDrive(double r, double robotAngle, double rightX) {
-        double v1 = r * Math.sin(robotAngle) - rightX;
-        double v2 = r * Math.cos(robotAngle) + rightX;
-        double v3 = r * Math.cos(robotAngle) - rightX;
-        double v4 = r * Math.sin(robotAngle) + rightX;
+        double v1 = -r * Math.sin(robotAngle) + rightX;
+        double v2 = -r * Math.cos(robotAngle) - rightX;
+        double v3 = -r * Math.cos(robotAngle) + rightX;
+        double v4 = -r * Math.sin(robotAngle) - rightX;
         setPower(v1,v2,v3,v4);
     }
 
+    public void autonDrive(double inches, double power, double robotAngle, double rightX) {
```

Comment:
this needs some rigorous testing.
rightX is negative for left turns, positive for right.